src/pseudopeople/interface.py

```diff
@@ -150,8 +150,8 @@ def generate_decennial_census(
     Generates a pseudopeople decennial census dataset which represents simulated
     responses to the US Census Bureau's Census of Population and Housing.
 
-    :param source: The root directory containing pseudopeople input data. Defaults
-        to the pseudopeople sample datasets directory.
+    :param source: The root directory containing pseudopeople simulated population data. Defaults
+        to the pseudopeople sample population directory.
```

Comment:
I know you didn't introduce this here, but I find this confusing, since it seems to contradict the fact that it actually defaults to `None`.

What it says at the top of the page is much clearer. Maybe something like this?

```suggestion
        to using the pseudopeople sample population.
```

Reply:
Or "to using the included sample population", or something else along those lines.

Reply:
Good point. I'll use your suggestion and update the other instances of this too.

Reply:
@zmbc I just changed it to "Defaults to using the included sample population when source is `None`." How does that sound?